File: baekjoon/silver/4134/4134.py
# input
# 3
# 6
# 20
# 100
# output
# 7
# 23
# 101

n = int(input())

def is_prime(num):
  if num == 0 or num == 1:
    return False
  for i in range(2, int(num**0.5)+1):
    if num % i == 0:
      return False
  return True

for i in range(n):
  s = int(input())
  while is_prime(s) == False:
    s += 1
  print(s)

File: baekjoon/silver/4134/4134.go
package main

import "fmt"

func main() {
	var t int

	fmt.Scan(&t)
	for i := 0; i < t; i++ {
		var n int

		fmt.Scan(&n)
		for isPrime(n) == false {
			n++
		}
		fmt.Println(n)
	}
}

func isPrime(n int) bool {
	if n == 0 || n == 1 {
		return false
	}
	for i := 2; i*i <= n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}
